practice/src/js/constants/message.js

```diff
@@ -0,0 +1,11 @@
+const MESSAGES = {
+  ADD_PRODUCT_SUCCESS_MSG: 'The dish was added successfully!',
+  ADD_PRODUCT_FAILED_MSG: 'The dish was added unsuccessfully!',
+  DELETE_PRODUCT_SUCCESS_MSG: 'The dish was deleted successfully!',
+  DELETE_PRODUCT_FAILED_MSG: 'The dish was deleted successfully!',
+  EDIT_PRODUCT_SUCCESS_MSG: 'The dish was edited successfully!',
+  EDIT_PRODUCT_SUCCESS_MSG: 'The dish was edited unsuccessfully!',
+  SEARCH_PRODUCT_LIST_EMPTY_HEADING: 'No dishes found!',
```

Comment:
You should use a common message here. For example:
Only use NOT_FOUND: 'No Results Found!'

This way, you can use this message in many places throughout the app, not just for product searches